sdk/core/Azure.Core/src/HttpMessage.cs

```diff
@@ -33,6 +33,7 @@ public HttpMessage(Request request, ResponseClassifier responseClassifier)
             Request = request;
             ResponseClassifier = responseClassifier;
             BufferResponse = true;
+            ProcessingStartTime = DateTimeOffset.UtcNow;
```

Comment:
Might be an edge case, but should we set this in the first policy in the pipeline instead? In theory the message could be created and not immediately sent/processed.

Reply:
Fair point - though I don't really like the idea of coupling a particular policy to the fact that it is the first in the pipeline. Users can always reset this if they want in a custom policy.

Reply:
We could set it in Pipeline.Send before starting the message down the pipeline if we wanted to ensure it got called at the very beginning without relying on policy ordering.

Reply:
Makes sense.